components/tree/Tree.razor.cs

```diff
@@ -324,7 +324,9 @@ public void CheckedAll()
             }
         }
 
-        // Decheck all of the checked nodes
+        /// <summary>
+        /// Unchecks all nodes.
+        /// </summary>
         public void DecheckedAll()
```

Comment:
Should we rename this method?

Reply:
I believe so:
https://github.com/ant-design-blazor/ant-design-blazor/pull/1790#issuecomment-888817711